### ISSUE TITLE
PlutoPDF doesn't print fully tables

### ISSUE THREAD

**Issue by ImreMD** [JuliaPluto/PlutoPDF.jl#15]
PlutoPDF seems to have a problem with printing full size table - on the pdf I have only few rows (the full tabled have between 50 - 90 rows - in the pdf only 
![pluto_table](https://github.com/JuliaPluto/PlutoPDF.jl/assets/35870578/aaf7926b-d37e-4c2b-a8a5-3ed77199a5c6)
![pdf_output](https://github.com/JuliaPluto/PlutoPDF.jl/assets/35870578/0651e8cc-e4ff-4eb7-bf95-c665e2b65349)

**Comment by ImreMD** [comment]
yes no problem it&#39;s financial data but it&#39;s publicly available so no worry.   I&#39;m adding a screenshot generated by the PlutoPDF    best regards,  Dnia 13 marca 2024 11:08 Fons van der Plas ***@***.***&gt; napisał(a):   Cann you share a notebook or html file?  —  Reply to this email directly,  github.com view it on GitHub , or  github.com unsubscribe .  You are receiving this because you authored the thread. Message ID:  &lt;JuliaPluto/PlutoPDF . jl/issues/15/1994015576 @ github . com&gt;

**Comment by fonsp** [comment]
I need a .jl notebook file or an .html export file.

**Comment by ImreMD** [comment]
Issue closed, yes? will have to find a workaround. Nevertheless thx for trying.    
     
      
       
        Dnia 20 marca 2024 10:02 Fons van der Plas ***@***.***&gt; napisał(a):
       
    
       
         
 Closed  github.com #15  as not planned.

**Comment by ImreMD** [comment]
I think that I know what&#39;s happening all large tables are exported to html with a automatic scrolling mechanism that&#39;s why when printed to pdf it&#39;s not printed fully , correcting this behavior should eliminate the issue.   best regards    
     
      
       
        Dnia 20 marca 2024 10:02 Fons van der Plas ***@***.***&gt; napisał(a):
       
    
       
         
 Closed  github.com #15  as not planned.
